common/types/activation_tx_header.go

```diff
@@ -10,6 +10,12 @@ type ActivationTxHeader struct {
 	// current epoch; a unit represents a configurable amount of data for PoST
 	NumUnits uint32
 
+	// EffectiveNumUnits is the minimum of this ATX's NumUnits and the previous ATX's NumUnits
+	// NumUnit decreases become effective immediately, while NumUnit increases become effective one epoch later
+	// This is because the increased PoST size only becomes effective after a PoET proof has
+	// been generated for it, which is published with the next epoch's ATX.
+	EffectiveNumUnits uint32
```

Comment:
Isn't `ActivationTxHeader` the type that gets gossiped/synced? I don't think we should send this value to others, they should calculate it themselves.

Reply:
`ActivationTxHeader` is not broadcasted to the network. It is only used by the node to not need to fetch and deserialize the whole ATX from the database and for caching.

Reply:
Ah, awesome then 👍🏻